### PR TITLE
tests.py: Add unit tests

### PR DESCRIPTION
**Pull request by jarifibrahim**
This fixes https://github.com/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/issues/69

### CODE REVIEW

**Comment by tapaswenipathak** [comment]
@jarifibrahim What all we can cover? #4 becomes more important now. This looks good to me, @amarlearning would you also like to review?

**Review by tapaswenipathak — APPROVED** [review]

File: oshc/main/tests.py [hunk 1]
```diff
@@ -1,6 +1,63 @@
...
+        response = self.client.login(username=self.credentials['email'],
+                                     password='incorrect_password')
+        self.assertFalse(response)
+
+
+class HomeViewTests(SimpleTestCase):
+
+    def test_get_request(self):
+        response = self.client.get(reverse("home"))
+        self.assertEqual(response.status_code, 200)
```
Comment (tapaswenipathak) [inline oshc/main/tests.py:61]: Should we also test for not success statuses?

Comment (jarifibrahim) [reply oshc/main/tests.py:61]: @tapasweni-pathak This test will raise an error if `home` responds with anything other than `200`. I don't think we need to test for not success statuses.

**Comment by jarifibrahim** [comment]
@tapasweni-pathak We don't have much code to test. These tests cover 
- Home view response status code i.e Main page is accessible
- Request session view response status code
- EmailLoginBackend - Test login with email instead of username

I'll send a PR for https://github.com/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/issues/4 soon.

**Comment by amarlearning** [comment]
@jarifibrahim Is it possible to add a test for registration form?

**Comment by jarifibrahim** [comment]
@amarlearning I don't think we need to write tests for the registration form. We're using a third party app for user registration. We trust that the developer has tested and then packaged the app. This [link](https://groups.google.com/d/msg/django-users/gGfVhfrfE10/k3CGtVoYcvsJ) answers why we shouldn't test third party apps.

**Pull request merged**